pyproject.toml

```diff
@@ -1,30 +1,25 @@
-[tool.black]
+[tool.ruff]
 line-length = 88
-target-version = ["py311"]
-include = '\.(wsgi|pyi?)$'
-exclude = '''
-/(
-    \.eggs
-  | \.git
-  | \.hg
-  | \.mypy_cache
-  | \.tox
-  | \.venv
-  | \.cache
-  | \.cache_py3
-  | _build
-  | buck-out
-  | build
-  | dist
-  | ui
-)/
-'''
+target-version = "py311"
+
+[tool.ruff.lint]
+select = [
+    "E", "W",       # pycodestyle
+    "F",            # pyflakes
+    "I",            # isort
+    "PLC", "PLE",   # pylint
+]
+ignore = [
+    "E402",
+    "E501",  # let ruff-format handle line-length
+    "E741",
+]
+
+[tool.ruff.lint.isort]
+known-first-party = ["ciadmin", "build_decision"]
+known-third-party = ["taskcluster", "tcadmin"]
+
 
 [tool.isort]
-multi_line_output = 3
-include_trailing_comma = true
-force_grid_wrap = 0
-use_parentheses = true
-line_length=88
```

Comment:
I think ruff does these settings by default. At least there's no equivalent to them and it still formatted our imports identically.